packages/dashboard/src/components/VisualizationsPanel/VisualizationsPanel.tsx

```diff
@@ -74,7 +82,19 @@ const VisualizationsPanel = () => {
   const [advancedEditing, setAdvancedEditing] = useState(false)
   const { config } = useContext(DashboardContext)
   const dispatch = useContext(DashboardDispatchContext)
-  const loadConfig = newConfig => dispatch({ type: 'APPLY_CONFIG', payload: [mapDataToConfig(newConfig)] })
+  const loadConfig = incomingConfig => {
+    const newConfig = !incomingConfig.multiDashboards
+      ? incomingConfig
+      : ({
+          ...config,
+          ...incomingConfig,
+          ...incomingConfig.multiDashboards[config.activeDashboard],
+          multiDashboards: incomingConfig.multiDashboards,
```

Comment:
isn't this already done on line 90?

Reply:
config has some properties that incomingConfig does not have.

Reply:
I'll say it another way: try deleting the multiDashboards declaration on line 92 and see if everything still works. I think you've already defined it on line 90 when you spread incomingConfig